tldw_Server_API/app/core/RAG/rag_service/rewrite_cache.py

```
@@ -84,12 +143,13 @@ def __init__(
         if path is None:
             if user_id:
-                base = Path("Databases/user_databases") / str(user_id) / "Rewrite_Cache"
                 try:
-                    base.mkdir(parents=True, exist_ok=True)
-                except Exception:
-                    pass
-                self.path = str(base / "rewrite_cache.jsonl")
+                    p = _resolve_user_cache_path(str(user_id))
+                    p.parent.mkdir(parents=True, exist_ok=True)
+                    self.path = str(p)
+                except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)